Last-Update: 2015-03-29
Forwarded: not-needed
Author: Dmitry Smirnov <onlyjob@member.fsf.org>
Description: "W: privacy-breach-generic" and the link is broken (404) anyway...

--- web/skins/flat/css/skin.css
+++ web/skins/flat/css/skin.css
@@ -20,10 +20,8 @@
 /*
  * Primary look and feel styles
  */
 
-@import url("https://fonts.googleapis.com/css?family=Open+Sans:200,200italic,300,300italic,400,400italic,600,600italic,700");
-
 body {
     font-family: "Open Sans", Verdana, Arial, Helvetica, sans-serif;
     font-size: 16px;
     color: #333333;
